src/main/resources/wa-task-permissions-wa-wacasetype.dmn

```diff
@@ -67,7 +67,7 @@ else
           <text>"tribunal-caseworker"</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_151ry1h">
-          <text>"Read,Refer,Own,Manage,Cancel"</text>
+          <text>"Read,Completeown,Cancelown,Claim,Own"</text>
```

Comment:
Completeown should be CompleteOwn
Cancelown should be CancelOwn

Reply:
corrected